index.html

```diff
@@ -309,9 +310,12 @@ <h1 class="h1 regular"><a class="underline" href="{{ post.url }}">{{ post.title
                 </div>
                 {% endfor %}
                 <h1 class="h1 regular">Older Announcements</h2>
-                  {% for post in site.posts offset:1 %}
+                  {% for post in site.posts offset:1 limit:5 %}
                   <h3 class="h3 regular"><a class="underline" href="{{ post.url }}">{{ post.title }}</a> <small>&mdash; {{ post.date | date: "%Y-%m-%d" }}</small></h1>
                 {% endfor %}
+                <div class="right-align mt2">
+                  <a href="{{site.baseurl}}/blog" class="h3 border-bottom">See all announcements</a>
```

Comment:
eventually would be good to change this url slug to something nicer, okay for the mo